packages/snaps-controllers/src/snaps/location/npm.ts

```diff
@@ -0,0 +1,353 @@
+import {
+  assertIsSemVerVersion,
+  assertIsSnapManifest,
+  deepClone,
+  DEFAULT_REQUESTED_SNAP_VERSION,
+  getTargetVersion,
+  isValidUrl,
+  NpmSnapIdStruct,
+  SemVerRange,
+  SemVerVersion,
+  SnapManifest,
+  VFile,
+} from '@metamask/snaps-utils';
+import { assert, assertStruct, isObject } from '@metamask/utils';
+import concat from 'concat-stream';
+import createGunzipStream from 'gunzip-maybe';
+import pump from 'pump';
+import { ReadableWebToNodeStream } from 'readable-web-to-node-stream';
+import { Readable, Writable } from 'stream';
+import { extract as tarExtract } from 'tar-stream';
+
+import { SnapLocation } from './location';
+
+const DEFAULT_NPM_REGISTRY = 'https://registry.npmjs.org';
+
+interface NpmMeta {
+  registry: URL;
+  packageName: string;
+  requestedRange: SemVerRange;
+  version?: string;
+  fetch: typeof fetch;
+}
+export interface NpmOptions {
+  /**
+   * @default DEFAULT_REQUESTED_SNAP_VERSION
```

Comment:
```suggestion
   * @default {@link DEFAULT_REQUESTED_SNAP_VERSION}
```

Reply:
`@default` values are treated as verbatim code, not possible.

<img width="924" alt="image" src="https://user-images.githubusercontent.com/1614945/204581035-17b2ebd4-355d-434d-bf03-592c2d5e5407.png">